cmd/ci-operator/main.go

```diff
@@ -263,9 +262,10 @@ func setupLogger() (*secrets.DynamicCensor, io.Closer, error) {
 	censor := secrets.NewDynamicCensor()
 	logrus.SetFormatter(logrusutil.NewFormatterWithCensor(logrus.StandardLogger().Formatter, &censor))
 	logrus.SetOutput(ioutil.Discard)
-	logrus.AddHook(&writer.Hook{
-		Writer: os.Stdout,
-		LogLevels: []logrus.Level{
+	logrus.AddHook(&formattingHook{
```

Comment:
That won't respect our censor formatter and we will end up leaking a secret. Can you check this or write a test that ensures that?

Reply:
Good point ...

Reply:
I remember I hit that issue before and I opened https://github.com/sirupsen/logrus/issues/1153. It worth taking a look at it.

Reply:
I don't think *that* particular bug will hit me, since I'm forcefully formatting in this code :|